languageFactories/wdl-biscayne/src/main/scala/languages/wdl/biscayne/WdlBiscayneLanguageFactory.scala

```diff
@@ -38,7 +38,7 @@ class WdlBiscayneLanguageFactory(override val config: Config) extends LanguageFa
 
     val checked: Checked[ValidatedWomNamespace] = for {
       _ <- enabledCheck
-      bundle <- getWomBundle(workflowSource, source.workflowOptions.asPrettyJson, importResolvers, factories)
+      bundle <- getWomBundle(workflowSource, None, source.workflowOptions.asPrettyJson, importResolvers, factories)
```

Comment:
I think this is a good example where naming variable would be helpful for the reader, i.e. `workflowSourceOrigin = None`

Reply:
Good point. Updated it 👍